tests/TestCase/ORM/TableTest.php

```diff
@@ -2997,33 +2997,30 @@ public function testDeleteCallbacks()
 
         $mock->expects($this->at(2))
             ->method('dispatch')
-            ->with($this->logicalAnd(
-                $this->attributeEqualTo('_name', 'Model.beforeDelete'),
-                $this->attributeEqualTo(
-                    '_data',
-                    ['entity' => $entity, 'options' => $options]
-                )
-            ));
+            ->with($this->callback(function (EventInterface $event) use ($entity, $options) {
+                return (
```

Comment:
phpcs doesn't like the use of parentheses for `return`.

https://github.com/cakephp/cakephp/pull/13382/checks?check_run_id=157743076